README.md

```diff
@@ -13,14 +18,20 @@ See [Data Transfer Project Overview](Documentation/Overview.md)
 * [Code of Conduct](CODE_OF_CONDUCT.md)
 
 ## Current State
-Data Transfer Project is in its early stages.. We are working with partners to define the architecture and implementation.  Please **DO NOT RUN THIS CODE
-IN PRODUCTION**, it has not had adequate security, privacy or
-productionisation testing or reviews done on it yet.
+Data Transfer Project is in its early stages, and we are actively looking for partner organizations and individuals to contribute to the project. We are continuing to define the architecture and implementation.  Since the code is in active development, please do thorough testing and verification before implementing. It has not been
```

Comment:
Consider removing "It has not been" fragment?

Reply:
Done. Thanks, good catch.